cognite/client/data_classes/contextualization.py

```diff
@@ -11,6 +11,21 @@
 from cognite.client.utils._auxiliary import convert_true_match
 
 
+class JobStatus:
+    QUEUED = "Queued"
+    RUNNING = "Running"
+    COMPLETED = "Completed"
+    FAILED = "Failed"
+    DISTRIBUTING = "Distributing"
+    DISTRIBUTED = "Distributed"
+    COLLECTING = "Collecting"
+    NOT_FINISHED_STATUSES = [QUEUED, RUNNING, DISTRIBUTED, DISTRIBUTING, COLLECTING]
```

Comment:
Use an enum instead?
```suggestion
class JobStatus(Enum):
    QUEUED = "Queued"
    RUNNING = "Running"
    COMPLETED = "Completed"
    FAILED = "Failed"
    DISTRIBUTING = "Distributing"
    DISTRIBUTED = "Distributed"
    COLLECTING = "Collecting"
    
    def is_finished() -> bool:
        return self in [JobStatus.QUEUED, JobStatus.RUNNING, JobStatus.DISTRIBUTED, JobStatus.DISTRIBUTING, JobStatus.COLLECTING]
```